diff -u abcde-2.1.15/abcde abcde-2.1.15_patched/abcde
--- abcde-2.1.15/abcde	2004-02-16 01:19:36.000000000 +0100
+++ abcde-2.1.15_patched/abcde	2004-02-22 11:58:10.003558520 +0100
@@ -808,6 +808,13 @@
 		CDDBLOCALSUCCESS="n"
 		CDDBDISCID=$(echo $TRACKINFO | cut -d' ' -f1)
 		CDDBLOCALFILE="${CDDBLOCALDIR}/${CDDBDISCID}"
+		# patched by tobi
+		# korrigiere die Suche nach einem lokalen Eintrag
+		t_CDDBLOCALFILE=$(find "${CDDBLOCALDIR}" -name "${CDDBDISCID}" -printf "%p  " | cut -d' ' -f1)
+		if [ "$t_CDDBLOCALFILE" != "" ]
+		then 
+		    CDDBLOCALFILE="$t_CDDBLOCALFILE"
+		fi
 		USELOCALCDDB="y"
 		
 		# If the user has selected to check a local CDDB repo, we proceed with it
@@ -1494,8 +1501,8 @@
 fi
 
 # Load system defaults
-if [ -r /etc/abcde.conf ]; then
-	. /etc/abcde.conf
+if [ -r /etc/abcde/abcde.conf ]; then
+	. /etc/abcde/abcde.conf
 fi
 # Load user preference defaults
 if [ -r $HOME/.abcde.conf ]; then
diff -u abcde-2.1.15/Makefile abcde-2.1.15_patched/Makefile
--- abcde-2.1.15/Makefile	2004-02-01 10:47:07.000000000 +0100
+++ abcde-2.1.15_patched/Makefile	2004-02-22 11:21:58.172726992 +0100
@@ -6,7 +6,7 @@
 exec_prefix = ${prefix}
 mandir = ${prefix}/share/man/man1
 bindir = ${exec_prefix}/bin
-etcdir = ${DESTDIR}/etc
+etcdir = ${DESTDIR}/etc/abcde
 
 all:
 
