--- ../abcde-2.1.19/abcde	2004-04-09 20:12:15.000000000 +0200
+++ abcde	2004-04-10 10:02:06.438658792 +0200
@@ -916,6 +916,13 @@
 		CDDBLOCALSUCCESS="n"
 		CDDBDISCID=$(echo $TRACKINFO | cut -d' ' -f1)
 		CDDBLOCALFILE="${CDDBLOCALDIR}/${CDDBDISCID}"
+                # patched by tobi
+                # korrigiere die Suche nach einem lokalen Eintrag
+                t_CDDBLOCALFILE=$(find "${CDDBLOCALDIR}" -name "${CDDBDISCID}" -printf "%p  " | cut -d' ' -f1)
+                if [ "$t_CDDBLOCALFILE" != "" ]
+                then
+                    CDDBLOCALFILE="$t_CDDBLOCALFILE"
+                fi
 		USELOCALRESP="y"
 		
 		# If the user has selected to check a local CDDB repo, we proceed with it
@@ -1703,8 +1710,8 @@
 fi
 
 # Load system defaults
-if [ -r /etc/abcde.conf ]; then
-	. /etc/abcde.conf
+if [ -r /etc/abcde/abcde.conf ]; then
+	. /etc/abcde/abcde.conf
 fi
 # Load user preference defaults
 if [ -r $HOME/.abcde.conf ]; then
