--- kfritz/CMakeLists.txt.orig	2010-02-16 14:49:38.000000000 +0100
+++ kfritz/CMakeLists.txt	2010-02-16 14:50:54.000000000 +0100
@@ -63,6 +63,7 @@
                    KSettingsFritzBox.cpp KFritzModel.cpp KCalllistModel.cpp LibFritzInit.cpp 
                    Log.cpp LogDialog.cpp QAdaptTreeView.cpp)
 set(CMAKE_CXX_FLAGS_DEBUG "-fexceptions -g -ggdb -O0 -Wall -D_GLIBCXX_DEBUG" ) 
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fexceptions")
 
 kde4_add_kcfg_files(kfritzbox_SRCS GENERATE_MOC KSettings.kcfgc)
 kde4_add_ui_files(kfritzbox_SRCS KSettingsFritzBox.ui KSettingsFonbooks.ui)
@@ -71,7 +72,7 @@
 kde4_add_executable(kfritz ${kfritzbox_SRCS})
 target_link_libraries(kfritz ${KDE4_KDEUI_LIBS} ${KDE4_KNOTIFYCONFIG_LIBRARY} 
                                 fritz++ tcpclient++ pthread++ 
-                                ssl)
+                                crypto ssl)
 if (INDICATEQT_FOUND)
 	target_link_libraries(kfritz ${INDICATEQT_LIBRARIES})
 endif (INDICATEQT_FOUND)
