--- kfritz/CMakeLists.txt.orig	2010-11-13 13:29:26.000000000 +0100
+++ kfritz/CMakeLists.txt	2010-11-13 13:29:46.000000000 +0100
@@ -76,7 +76,7 @@
 configure_file(pkg-config.h.cmake ${CMAKE_CURRENT_BINARY_DIR}/pkg-config.h)
 kde4_add_executable(kfritz ${kfritzbox_SRCS})
 target_link_libraries(kfritz ${KDE4_KDEUI_LIBS} ${KDE4_KNOTIFYCONFIG_LIBRARY} ${GCRYPT_LIBRARIES}
-                                fritz++ tcpclient++ pthread++ )
+                                fritz++ tcpclient++ pthread++ gcrypt)
 if (INDICATEQT_FOUND)
 	target_link_libraries(kfritz ${INDICATEQT_LIBRARIES})
 endif (INDICATEQT_FOUND)
